src/credentials/activation.ts

```diff
@@ -32,9 +32,11 @@ export async function loginWithMostRecentCredentials(
     toolkitSettings: SettingsConfiguration,
     loginManager: LoginManager
 ): Promise<void> {
-    const profiles = await CredentialsProviderManager.getInstance().getCredentials()
+    const manager = CredentialsProviderManager.getInstance()
+    const profiles = await manager.getCredentials()
```

Comment:
Looking at this file's changes, I was confused until I looked at getCredentials. I think we had a miss in the previous PR.
`getCredentials` should be `getCredentialsProviderIds`, because it isn't returning credentials. 

Also: suggest renaming `profiles` to more accurately reflect what it represents.

Reply:
`CredentialsProviderId.asString()` returns a profile name, no?

Reply:
Think of "profile" as an implementation detail of "Shared Credentials". "Shared Credentials" is an implementation (or type) of Credentials. There are other Credentials types... you can see examples of these in `defaultProviders` in https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/CredentialProviderChain.html

See https://github.com/aws/aws-toolkit-vscode/blob/master/designs/credentials/credentials-management.md#credentials-provider-id for information on CredentialsProviderId

Reply:
still have this question: can you describe what `CredentialsProviderId.asString()` returns?

Reply:
It returns a formatted-for-presentation (eg: what you see in the statusbar, or when selecting profiles to connect with) representation of a CredentialsProvider instance, which includes information relating to its credentials type, as well as instance identifying information.

For Shared Credentials (credentials coming from the ~/.aws/credentials file), each instance represents a profile from that file, and asString makes this into "profile:<profilename>".

Imagine if the Toolkit had an EC2 CredentialsProvider. There wouldn't be different instances of this, and there are no concepts of profile here. asString would probably return something along the lines of "ec2:instance"

Reply:
great, I'll add that as a docstring to `asString()` and now I can rename the function that uses it.